rollup-bridge-contracts/l2-blob-proof-playground/readme.MD

```diff
@@ -0,0 +1,237 @@
+# DataProof Integration Guide
```

Comment:
if all `l1-blob-proof-playground` is example of generate DA + kzg proof, let's get rid whole folder

Reply:
to be deleted soon